modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -138,6 +138,7 @@ The `AllowedSubjectPatterns` is an optional value that specifies a list of regul
 *** `syslog` describes parameters for the `Syslog` logging destination type:
 **** `address` is the IP address of the syslog endpoint that receives log messages.
 **** `port` is the UDP port number of the syslog endpoint that receives log messages.
+**** `maxLength` is the maximum length of the syslog message. It must be between 480 and 4096 bytes. The maximum length is 1024 bytes by default. If this field is empty, the maximum length is set to the default value.
```

Comment:
I think this would be more useful if you formatted the range and default values as values, like in line 142. Consider also combining the last two sentences, sort of like:  

```
If this field is empty, the maximum length is set to the default value of `1024` bytes.
```

Reply:
@kalexand-rh Comment addressed.